tests/integration/suite/test_catalog.py

```diff
@@ -26,6 +27,55 @@ def test_catalog(admin_mc):
     wait_for_template_to_be_deleted(client, name2)
 
 
+def test_invalid_catalog(admin_mc):
+    client = admin_mc.client
+    name = random_str()
+    bad_url = "git://github.com/StrongMonkey/charts-1.git"
+    # POST: Bad URL
+    with pytest.raises(ApiError) as e:
+        client.create_catalog(name=name,
```

Comment:
save the catalog and pass it to the `remove_resource` fixture in case it actually does get created.